### DISCUSSION TITLE
Monthly Community Call of the Single Source Publishing Community

### DISCUSSION THREAD

**Discussion by xldrkp** [singlesourcepub/community#34] [Community Calls]
Be ready to meet interesting people and to discuss cool stuff in the new **monthly community call**! No other preparation needed - we all are experts who can give an instant talk on their topics even in the middle of the night ;-) Don't be shy, we know you can ...

### When?

| Day | Date | Time (CET) |  |
| --- | --- | --- | --- |
| Wednesday | 10.01.2024 | 12:00 pm | 30 min |
| Thursday | 01.02.2024 | 12:00 pm | 30 min |
| Friday | 01.03.2024 | 12:00 pm | 30 min |
| Thursday | 04.04.2024 | 12:00 pm | 30 min |

Yes, we know, it's lunch time. But as it was OK for many people to meet at noon when we asked in October, so we want to give it a try again.

**If you want to be notified in advance, please let us know with a mail to info@singlesource.pub.**

### Where?

We can provide a cozy meeting room in Jitsi that you can find right around the corner at https://meet.jit.si/singlesourcepublishing. There will be no recording, just written notes on what was discussed.

### Why attend?

- Exchange about new tools and technologies.
- Show and tell what you are doing. 
- Inspire others with your work.
- Meet the people doing similar things you do.
- Discover funding and calls for proposals.
- Join forces to collaborate more and compete less.
- Shape independent and open ways of scholarly publishing. 

Looking forward to chatting with you!

**Comment by tarleb** [comment]
Note on the ambiguity of the given times: all meeting are at noon in the Central Europe time zone. This is currently UTC+2 (and UTC+1 in winter). E.g., the ISO time for the upcoming April meeting is 2022-04-06T10:00:00Z.

**Comment by castedo** [comment]
Ooops, I forgot to save the link to the survey/questionnaire in today's meeting. Can someone send it to me if it's not too late?

**Reply by tarleb** [reply]
Here's the link: https://cloud.nfdi4culture.de/apps/forms/rnGNMXYMoZm5JKZr

**Reply by mrchristian** [reply]
here is some more context https://github.com/singlesourcepub/community/discussions/44

**Comment by cderv** [comment]
Unfortunately, I won't be able to make it for today community call. If anything related to some of my projects, please fill free to contact me after the meeting.

**Comment by mrchristian** [comment]
Hi peeps, will join you at 12:00 noon CEST today for the community call. I wanted to share an invite to a workshop on Sematic Enrichment of IPCC reports - https://github.com/petermr/semanticClimate

**Comment by castedo** [comment]
I can join and can talk about an experimental system I recently start trying out, if anybody is interested to hear about it. Here's a description https://popgen.es/tech/ and two example documents/articles are:
* https://popgen.es/DZFCt68peNNajZ34WtZni9VYxzo/0.3
* https://popgen.es/0iV47kWzQAuyONrIDG538k3x3Qc/0.1

**Comment by castedo** [comment]
During this month's call I did not have very much written up about the experimental system I presented. I have now posted two documents which provide much more detail:

1. a non-technical document about the [perm.pub pilot project](https://perm.pub/czdv8PyJKF7LneTnaVT6pgAKyh8/0) and
2. a technical specification of [Digital Succession Identifiers](https://perm.pub/ji2STto1mZ3i2BmnGxbkebejKH4/0).

Both documents are themselves examples of digital article successions, of course :smile:!

If anybody is interested in being part of the pilot project, has feedback or suggestions, let's discuss! Feel free to use the github discussions feature here for feedback, or email me directly.

**Comment by xldrkp** [comment]
Morning! I suggest to spend some time talking about https://kotahi.community/ in our next call. Has anybody tried it yet? I know it for long and spoke to Adam years ago. Now I wonder why we never took this approach into account for our work. Ideas?

**Reply by ronste** [reply]
This look interesting. I never tried this platform myself.

To add some context:

In a [survey of the European Commission](https://op.europa.eu/en/publication-detail/-/publication/cc087fd8-82b3-11ee-99ba-01aa75ed71a1/language-en) completed in April 2023 Kothai gained the highest score in the category "Publication" (see page 7).
